run.py

```diff
@@ -384,8 +385,52 @@ def initrands (s=0): # fix to use s
 
 def setupLFPelectrodes ():
   lelec = []
+
+  # define locations in pixels (microns) of HNN model features
+  top_l5 = 1466
+  bottom_l5 = -72
+
+  # define laminar probe properties
+  num_contacts = 20
+  contact_spacing = (top_l5 - bottom_l5)/(num_contacts - 5.0)
```

Comment:
If you have spent some effort in figuring this out, perhaps it could be worth porting this to mne-neuron/hnn-core -- in a cleaner way. I know it doesn't feel like it's worth the effort but keeping things clean will pay off eventually I promise :)